### PR TITLE
[DOCS] New "Getting Started" landing page

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/1835.

Previews:
- [Elastic Security UI | Getting started page](https://security-docs_1858.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#getting-started-page)
- [Ingest data to Elastic Security](https://security-docs_1858.docs-preview.app.elstc.co/guide/en/security/master/ingest-data.html)

### CODE REVIEW

**Review by stephmilovic — APPROVED** [review]
LGTM! Thanks for the updates @nastasha-solomon :)

File: docs/getting-started/security-ui.asciidoc [hunk 1]
```diff
@@ -50,6 +51,14 @@ TIP: Click the *Collapse side navigation*  button to collapse and expand the mai
 [role="screenshot"]
 image::images/collapse-side-nav-button.gif[width=100%][height=100%][Demo that shows how to to scroll through the page]
 
+[float]
+[[getting-started-page]]
+=== Getting started page
+
+The Getting started page guides you to integrations that you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the getting started prompt until event data is available.
```
Comment (benironside) [inline docs/getting-started/security-ui.asciidoc:58]:
```suggestion
The Getting started page guides you to integrations that you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the getting started prompt until event data has been ingested.
```

**Review by benironside — APPROVED** [review]
LGTM, left one minor suggestion.

**Review by joepeeples — APPROVED** [review]
LGTM!

**Review by jmikell821 — APPROVED** [review]
LGTM!

**Pull request merged**